source/material_model/rheology/drucker_prager.cc

```diff
@@ -181,6 +184,33 @@ namespace aspect
 
 
 
+      template <int dim>
+      std::pair<double,double>
+      DruckerPrager<dim>::compute_dilation_terms_for_stokes_system(const double angle_friction,
+                                                                   const double angle_dilation,
+                                                                   const double cohesion,
+                                                                   const double non_yielding_viscosity,
+                                                                   const double effective_strain_rate) const
+      {
+        const double sin_phi = std::sin(angle_friction);
+        const double sin_psi = std::sin(angle_dilation);
+
+        double alpha     = sin_phi;
+        double alpha_bar = sin_psi;
+        if (dim == 3)
+          {
+            alpha     = 6.0 * sin_phi / (std::sqrt(3.0) * (3.0 + sin_phi));
+            alpha_bar = 6.0 * sin_psi / (std::sqrt(3.0) * (3.0 + sin_psi));
+          }
```

Comment:
Let's be more explicit about this:
```suggestion
        double alpha;
        double alpha_bar;
        if (dim == 2)
        {
          alpha     = sin_phi;
          alpha_bar = sin_psi;
        }
        else if (dim == 3)
          {
            alpha     = 6.0 * sin_phi / (std::sqrt(3.0) * (3.0 + sin_phi));
            alpha_bar = 6.0 * sin_psi / (std::sqrt(3.0) * (3.0 + sin_psi));
          }
        else
          AssertThrow(false,ExcNotImplemented());
```